frontend/src/v4/modules/store.ts

```diff
@@ -22,6 +22,7 @@ import { applyMiddleware, compose, createStore } from 'redux';
 import createSagaMiddleware from 'redux-saga';
```

Comment:
I am looking at the same drawings list from 2 different users (A and B). User A changes the latest active revision. For as long as that drawing is not expanded for User B (i.e. for as long and they haven't expanded the revisions list), the revisions updates via WS are not considered 
 
https://github.com/user-attachments/assets/68840cc7-3d4e-4269-a78d-a8a6fadcd040

The same behaviour is observable for containers